packages/cli/src/commands/init.ts

```diff
@@ -0,0 +1,41 @@
+
+import fs from 'node:fs';
+import degit from 'degit';
+import { Command } from '@oclif/core';
+import { confirm } from '@inquirer/prompts'
+
+export default class Init extends Command {
+  static args = [
+    {
+      name: 'path',
+      description: 'The path where the Discovery folder will be created. Defaults to ./discovery.'
+    }
+  ]
+
+  static description = "Creates a discovery folder based on the starter.store template.";
+
+  static examples = ["$ yarn faststore init discovery"];
+
+  async run() {
+    const { args } = await this.parse(Init)
+
+    const discoveryPath = args.path ?? "./discovery"
+    const discoveryFolderExists = fs.existsSync(discoveryPath);
+
+    if (discoveryFolderExists) {
+      const confirmOverride = await confirm({ message: 'It looks like you already have a discovery folder in your store. Do you want to override it?' });
```

Comment:
suggestion: to make it clear which folder is having the issue and will be overridden, what do you think?
```suggestion
      const confirmOverride = await confirm({ message: `It looks like you already have a discovery folder named "${discoveryPath}" in your store. Do you want to override it?` });
```

Reply:
@lariciamota that makes sense! Thanks for the suggestion.